code/pizzabot.js

```diff
@@ -1,7 +1,89 @@
-const vegetarian = "Vegetarian Pizza"
-const hawaiian = "Hawaiian Pizza"
-const pepperoni = "Pepperoni Pizza"
+const vegetarian = "Vegetarian Pizza";
+const hawaiian = "Hawaiian Pizza";
+const pepperoni = "Pepperoni Pizza";
 
-const pizzaPrice = 80
+const pizzaPrice = 80;
+
+let botSays = document.getElementById('botSays');
+botSays.innerHTML = 'What would you like to order?';
+
+// make inputbox actice (the mouse/cursor is there in beginning)
+document.getElementById("answer").focus();
+
+// used to check if valid pizza is ordered
+let orderedPizza = "";
+
+const isValidPizza = (pizza) => {
+    if (vegetarian.toLowerCase() === pizza.toLowerCase() ||
+        hawaiian.toLowerCase() === pizza.toLowerCase() ||
+        pepperoni.toLowerCase() === pizza.toLowerCase()) {
+        return true
+    } else {
+        return false
```

Comment:
Regarding clean code, there is also another convention, let me introduce you to it.
Apart from returning given thing from function, `return` keyword does one more thing - it automatically finishes function. That's why we mustn't write anything below return, because that code will never be executed (of course in your case we have code below return true, but it's inside if statement, so it wouldn't enter second statement in the first place).
Thanks to that feature of `return` keyword, we can write our function in slightly different, far more clean fashion :
```
const validatePizza = pizza => {
    if (vegetarian.toLowerCase() === pizza.toLowerCase() ||
        hawaiian.toLowerCase() === pizza.toLowerCase() ||
        pepperoni.toLowerCase() === pizza.toLowerCase()) {
        return true
    }
    return false
}
```
As you can see, we are not writing `else` statement anymore. We can do that thanks to behaviour mentioned above; if result of if conditional is `true`, we will execute code from line 18 onwards and finish function at line 20. If, on the other hand, result of conditional was `false`, we will never enter line 18 and jump straight into line 22. After executing line 22 we will finish function same way :)